docs/docs/reference/release-notes/v3.7/index.md

```diff
@@ -0,0 +1,109 @@
+
+This new API will be used to write out information that ties the state of some CMS content to a finally built page.
+Useful for routing to previews or even production content once they're built.
+
+This release adds a new public action `unstable_createNodeManifest` which is used to tie a manifest ID
+(a CMS id that maps to a unique revision state of some content) to a page that was built from a specific node.
+
+To make the mapping from node to page more accurate, this release introduces a new argument to the `createPage` helper,
+`ownerNodeId` so that a user can specify that a page is owned by a specific node.
+
+In the case that no `ownerNodeId` is provided, the logic checks for a page with an `id` variable in page context that
+matches to the node id of the node manifest. If neither of those exist the logic maps the manifest to the first
+page the node is found on in query tracking.
+
+The result is that a source plugin can allow a CMS to create a manifest file using a CMS ID that maps to a finally built page in Gatsby, allowing for a service to redirect to the right page after a build is complete.
+
+[Original PR](https://github.com/gatsbyjs/gatsby/pull/31127).
+
+## Experimental: Node persistence in LMDB
+
+This release introduces a new experimental data storage option: LMDB (via the excellent [lmdb-store](github.com/DoctorEvidence/lmdb-store) package).
+
+Instead of keeping nodes in memory (as it's in Redux), they are instantly saved to this persistent embeddable storage.
+This will be beneficial for larger sites that can run into OOMs when persisting redux state to disk.
+Several other performance improvements using this new data store will follow.
+
+We encourage you to try it and [let us know](https://github.com/gatsbyjs/gatsby/discussions/31769) if you encounter any issues!
+
+Installation instructions are in the [Umbrella Discussion](https://github.com/gatsbyjs/gatsby/discussions/31769).
+
+[Original PR](https://github.com/gatsbyjs/gatsby/pull/31371)
+
+## gatsby-remark-images: async image decoding by default
+
+Added new plugin option `decoding` that adds corresponding attribute to all `img` tags produced by the plugin.
+Default value is `async` (other allowed values are: `sync` or `auto`).
+
+The `decoding` property allows you to control if the browser is allowed to try to parallelize loading your image.
+If doing so would cause problems, you can specify `sync` to disable asynchronous loading.
+
+[Original PR](https://github.com/gatsbyjs/gatsby/pull/31558)
+
+[Read more about the `decode` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding).
+
+## Yarn 2 (PNP) support
+
+Yarn 2 with Plug'n'Play (Pnp) is available again. With the upgrade of Webpack v4-v5 during our Gatsby v3 release, PnP support broke. We've added an e2e-test to make sure this won't happen again in the future. With yarn 2, you'll notice faster NPM installs and less data usage as yarn2 is aggresivily caching node_modules.
```

Comment:
Does that refer to https://github.com/gatsbyjs/gatsby/pull/31749?

Reply:
yes